pkg/operator/vitesstopo/prune_cells.go

```diff
@@ -82,7 +82,7 @@ func DeleteCells(ctx context.Context, ts *topo.Server, recorder record.EventReco
 
 	for _, cellName := range cellNames {
 		// topo.NoNode is the error type returned if we can't find the cell when deleting. This ensures that this operation is idempotent.
-		if err := ts.DeleteCellInfo(ctx, cellName); err != nil && !topo.IsErrType(err, topo.NoNode) {
+		if err := ts.DeleteCellInfo(ctx, cellName, true); err != nil && !topo.IsErrType(err, topo.NoNode) {
```

Comment:
```suggestion
		if err := ts.DeleteCellInfo(ctx, cellName, true /*force*/); err != nil && !topo.IsErrType(err, topo.NoNode) {
```

Reply:
We should leave this as `false` because that was the behavior before the new argument was added.